babashka.el

```diff
@@ -50,51 +76,66 @@
      (insert-file-contents file-path)
      (buffer-string))))
 
-(defun babashka--run-shell-command-in-directory (directory command &optional output-buffer)
+(defun babashka--run-shell-command-in-directory (directory command)
```

Comment:
Also, I don't see any usage of `output-buffer` arg here, so I decided to remove it for unification